_format/1.0/index.md

```diff
@@ -639,6 +640,15 @@ The following characters **MUST NOT** be used in member names:
 Data, including resources and relationships, can be fetched by sending a
 `GET` request to an endpoint.
 
+In general, servers **SHOULD** service fetching via a `GET` request. However, in cases
+where it is possible that lengthy search parameters provided via query parameters 
+may exceed the maximum URL length (1024), servers **MAY** support fetching via 
+a `POST` request. In that case the search parameters **MUST** be included in the POST body.
+
+In such a case, where a fetch is supported via a `POST` request, the server **SHOULD NOT** support
+fetch via both `GET` and `POST`; rather only `POST` may be supported. In addition, this
+deviation from the norm **MUST** be clearly called out in the supporting documention (e.g. OpenAPI).
```

Comment:
My hope is that fetching via POST would be limited to things like custom reports, where users could be passing in hundreds of IDs to generate a report. For simple search and filtering of collections (e.g. outbound, inbound, inventory, etc), I do not think we should support fetching via POST.

Reply:
I can call that out. Good point.

Reply:
That preference for GET already seemed clear to me by the first paragraph. What else exactly is being called out?

Reply:
2 points:

* dont use both GET and POST
* document

Reply:
yes, we did, to be clear. We dont want that to be a hard requirement where it doesnt make sense.